### PR TITLE
About page

### PR DESCRIPTION
**Pull request by RobbieZhuang**
Built About Page Desktop + Mobile #4 and moved images to `/public` folder #30 

**Desktop**
Screen Shot 2020-05-09 at 6 45 13 PM

**Mobile**
Screen Shot 2020-05-09 at 6 45 26 PM

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,14 +1,15 @@
 import React, {Component} from "react";
 import {BrowserRouter as Router, Route, Switch} from "react-router-dom";
+import url from "url";
 import "./App.scss";
 import Home from "./containers/Home.jsx";
-import Platforms from "./containers/Platforms.jsx";
```
Comment (RobbieZhuang) [inline src/App.jsx:5]: Removed outdated platforms page from the path.

File: src/containers/About.jsx [hunk 2]
```diff
@@ -1,183 +1,93 @@
...
+            {
+              name: "Roxane Fruytier",
+              src: profileImage("roxane"),
+              year: "2021",
+              position: "President",
+            },
+            {
+              name: "Elisa Luan",
+              src: profileImage("elisa"),
+              year: "2021",
```
Comment (jenniferwills) [inline src/containers/About.jsx]: Elisa and Mayank are 2023

File: src/containers/About.jsx [hunk 3]
```diff
@@ -1,183 +1,93 @@
...
+              position: "VP Events",
+            },
+            {
+              name: "Mayank Kanoria",
+              src: profileImage("spencer"),
+              year: "2021",
```
Comment (jenniferwills) [inline src/containers/About.jsx]: 2023

File: src/containers/About.jsx [hunk 4]
```diff
@@ -1,183 +1,93 @@
...
+        </p>
+        <h1>Meet The Team</h1>
+        <p>
+          B-Soc represents the odd years (SE2021, SE2023, etc.) and A-Soc
+          represents the even years (SE2020, SE2022, etc.). We’re a friendly
+          bunch, feel free to say hi if we ever bump into each other on campus!
+        </p>
+        <ProfilePhotoGroup
+          soc={"B-Soc"}
+          execs={[
```
Comment (jenniferwills) [inline src/containers/About.jsx]: Is the expected format for info on the social reps the same as the execs? Do we need to say ```position: "social rep"``` for all the reps as well? (the year they represent would also be shown from their own year, so the rep position text would be the same across all of the reps)

File: src/containers/About.jsx [hunk 5]
```diff
@@ -1,183 +1,93 @@
...
+            },
+            {
+              name: "Elisa Luan",
+              src: profileImage("elisa"),
+              year: "2021",
+              position: "VP Events",
+            },
+            {
+              name: "Mayank Kanoria",
+              src: profileImage("spencer"),
```
Comment (jenniferwills) [inline src/containers/About.jsx]: oops guess we're still waiting on Mayank. (Is it too late to ask Ariel for a cute place holder character to use when we don't have picture of someone, or they would rather not have a picture on the website?)

File: src/containers/About.jsx (hunk 2 above)
Comment (RobbieZhuang) [reply src/containers/About.jsx]: Oops, thanks for letting me know :)

File: src/containers/About.jsx (hunk 5 above)
Comment (RobbieZhuang) [reply src/containers/About.jsx]: That's a great idea, it's never too late! I can ask ^^

File: src/containers/About.jsx (hunk 4 above)
Comment (RobbieZhuang) [reply src/containers/About.jsx]: I agree year and position would be a little repetitive for reps. 
For consistency, we could keep `year` and set position as `1A Rep, 3B Rep, etc.`

**Pull request merged**